### PR TITLE
feat(helm): expose service type value

### PR DESCRIPTION
**Pull request by sl1pm4t**
This adds support for specifying the service type in helm values.

### CODE REVIEW

File: helm/cluster-api-visualizer/values.yaml [hunk 1]
```diff
@@ -8,3 +8,6 @@ image:
 label: 
   key: app
   value: capi-visualizer
+
+service:
+  type: ClusterIP
```
Comment (Jont828) [inline helm/cluster-api-visualizer/values.yaml:13]: Could you add a newline?

**Review by Jont828 — CHANGES_REQUESTED** [review]
Thanks for the contribution! Just one minor nit and we're good to go. @sl1pm4t

File: helm/cluster-api-visualizer/values.yaml (hunk 1 above)
Comment (sl1pm4t) [inline helm/cluster-api-visualizer/values.yaml:13]:
```suggestion
  type: ClusterIP

```

**Review by Jont828 — APPROVED** [review]

**Comment by Jont828** [comment]
Thanks!

**Pull request merged**